modules/cluster-logging-about-collector.adoc

```diff
@@ -15,7 +15,7 @@ By default, the log collector uses the following sources:
 
 If you configure the log collector to collect audit logs, it gets them from `/var/log/audit/audit.log`.
 
-The logging collector is a daemon set that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}. Application logs are generated by the CRI-O container engine. Fluentd collects the logs from these sources and forwards them internally or externally as you configure in {product-title}.
+The logging collector is a daemon set that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}. Application logs are generated by the CRI-O container engine. The collector collect the logs from these sources and forwards them internally or externally as you configure in {product-title}.
```

Comment:
Verb tense 

`The collector collects the logs from these sources and forwards them internally or externally as you configure in {product-title}.`